plenoptic/synthesize/mad_competition.py

```diff
@@ -1697,6 +1699,11 @@ def _grab_value_for_comparison(self, value, iteration=None,
         ----------
         value : {'representation', 'signal'}
             Whether to compare the representations or signals
+        batch_idx : int, optional
+            Which index to take from the batch dimension (the first one)
```

Comment:
Remove "the first one" or rewrite to something more explicit bc i thought it meant first one is default idx (and it is, since batch_idx=0)

Reply:
done